src/test/java/com/rackspace/salus/policy/manage/web/controller/MonitorPolicyApiControllerTest.java

```diff
@@ -107,6 +109,8 @@ public void testGetById() throws Exception {
   public void testGetEffectivePoliciesByTenantId() throws Exception {
     String tenantId = RandomStringUtils.randomAlphabetic(10);
     final List<MonitorPolicy> listOfPolicies = podamFactory.manufacturePojo(ArrayList.class, MonitorPolicy.class);
+
+    List<MonitorPolicyDTO> listOfPoliciesDTO = listOfPolicies.parallelStream().map(MonitorPolicyDTO::new).collect(Collectors.toList());
```

Comment:
Any particular reason for parallel stream?
```suggestion
    List<MonitorPolicyDTO> listOfPoliciesDTO = listOfPolicies.stream().map(MonitorPolicyDTO::new).collect(Collectors.toList());
```

Reply:
just thought of performance.

Reply:
In a unit test with a collection of I think 5 elements it isn't worth the thread pool overhead. No need to change this, but always be cautious of pre-optimizing code.